tests/integration.rs

```diff
@@ -149,6 +151,7 @@ fn mount(harness: Harness) {
 	drop(harness);
 }
 
+// TODO: find all files recursively
```

Comment:
I disagree.  The expected list of files should be hard-coded into integration.rs, and should be updated whenever mkimg.sh is.  Otherwise, if you try to find files from within integration.rs, the test will wrongly fail to detect bugs in readdir that omit files.

Reply:
By "find all files recursively" I meant to replace the single `readdir()` with a full traversal of the mount.
The list of expected files will remain hard-coded, but will be expanded.